mlflow/store/db_migrations/versions/d2999357da94_change_param_value_length_to_500.py

```diff
@@ -0,0 +1,34 @@
+"""change param value length to 500
+
+Revision ID: d2999357da94
+Revises: bd07f7e963c5
+Create Date: 2022-07-27 22:00:13.187596
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "d2999357da94"
+down_revision = "bd07f7e963c5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """
+    Enlarge minimum length of `value` columns to 500.
```

Comment:
Are we sure this line is correct? We increase the maximum value length, not minimum, right?

Reply:
thanks, will fix